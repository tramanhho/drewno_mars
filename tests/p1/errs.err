FATAL [1,1]-[1,2]: Illegal character %
FATAL [1,2]-[1,3]: Illegal character @
FATAL [2,1]-[2,7]: String literal with bad escape sequence detected
FATAL [3,1]-[3,12]: Unterminated string literal detected
FATAL [4,1]-[4,6]: Unterminated string literal with bad escape sequence detected
