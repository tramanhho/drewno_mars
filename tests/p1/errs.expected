EOF [4,6]